docs/en/core/math.md

```diff
@@ -1,9 +1,23 @@
 # Math
 
-Zettlr supports writing math and formulae using the KaTeX library. In order to use math, enclose your formulae with dollar signs (`$`). You can also use so-called "display math" by using two dollar signs instead of single dollar signs to enclose your formulae.
+Zettlr supports writing math and formulae using the KaTeX library. In order to use math, enclose your formulae with dollar signs (`$`). You can also use math blocks by using two dollar signs instead of a single dollar sign to enclose your formulae. On export, the double dollar signs are replaced with the LaTeX environment commands `\begin{equation}` and `\end{equation}`. Anything detected as inline or math block will receive proper MathTeX syntax highlighting.
```

Comment:
I would keep "display math" in here, since that's kind of an important term, so it might be good if people are searching online, and then search for that term here.

Plus, these two dollar signs will only be replaced with TeX commands if you export to TeX. In HTML exports, they will be replaced with math tags which are then rendered by MathTeX.